.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
```

Comment:
For this job, replace the `setup.py` with `pip install -e .[test]` (`-e` for editable/in-place, `[test]` for test optional dependencies). With that you can remove the `setup.py` and all of the `requirements-*.txt`